crates/cargo-codspeed/src/run.rs

```diff
@@ -143,5 +166,54 @@ pub fn run_benches(metadata: &Metadata, filters: Filters) -> Result<()> {
         eprintln!("Done running {}", bench_name);
     }
     eprintln!("Finished running {} benchmark suite(s)", to_run.len());
+
+    if measurement_mode == MeasurementMode::Walltime {
+        aggregate_raw_walltime_data(workspace_root)?;
+    }
+
+    Ok(())
+}
+
+fn clear_raw_walltime_data(workspace_root: &Path) -> Result<()> {
+    let raw_results_dir = get_raw_result_dir_from_workspace_root(workspace_root);
+    std::fs::remove_dir_all(&raw_results_dir).ok(); // ignore errors when the directory does not exist
+    std::fs::create_dir_all(&raw_results_dir).context("Failed to create raw_results directory")?;
+    Ok(())
+}
+
+fn aggregate_raw_walltime_data(workspace_root: &Path) -> Result<()> {
+    let mut walltime_benchmarks: Vec<WalltimeBenchmark> = Vec::new();
+    // retrieve data from `{workspace_root}/target/codspeed/raw_results/{scope}/*.json
+    for sample in glob(
+        get_raw_result_dir_from_workspace_root(workspace_root)
+            .join("**/*.json")
```

Comment:
Very minor, but wouldn't it make more sense to push `**/*.json` after the stringified path rather than doing it the other way around? Not sure if `**/*.json` makes sense in a path object